tests/scripts/thread-cert/pktverify/utils.py

```diff
@@ -135,3 +135,22 @@ def which_mergecap() -> str:
     :return: The path to `mergecap` executable.
     """
     return os.path.join(get_wireshark_dir(), 'mergecap')
+
+
+def ridmask_to_rid(ridmask: str) -> list:
+    """
+    Convert router ID mask to router IDs
+
+    :param ridmask: router ID mask
+    :return: router id list
+    """
+    assert isinstance(ridmask, str), ridmask
+    ridmask_int = int(ridmask.replace(':', ''), base=16)
```

Comment:
is there `:` in `ridmask`?

Reply:
yes, from wireshark filter, the router id mask format likes "mle.tlv.route64.id_mask == 00:40:00:00:80:00:00:00"

Reply:
Noted. Thanks.
I may suggest to add note to improve the readability in the future.